sklearn/datasets/tests/test_samples_generator.py

```diff
@@ -242,6 +242,8 @@ def test_make_hastie_10_2():
 
 
 def test_make_regression():
+    pd = pytest.importorskip('pandas')
+    np = pytest.importorskip('numpy')
```

Comment:
Numpy is a hard dependency of scikit-learn, no need to use `pytest.importorskip`, just use a regular top-level import if not already imported.

Reply:
already imported, I'll remove the line. Thanks!